definitions/json/CaseField/CaseField-Bundles.json

```diff
@@ -2,14 +2,14 @@
   {
     "CaseTypeID": "ET_EnglandWales",
     "ID": "bundlesRespondentPrepareDocNotes1",
-    "Label": "The tribunal should have told you in a letter which documents you need to submit for your hearing.<br><br>These documents, usually agreed by both parties, will form part of the evidence at the hearing. You should include all the documents that are important to your case, and that both parties want to refer to at the hearing.<br><br>The combined set of documents must be in one PDF file, usually submitted by one party unless instructed otherwise by the tribunal.<br> <br>If an index is requested, items should be in date order or an order in which the tribunal has requested.<br><br>The single PDF document should be created offline before uploading through this form.<br><br>You must submit hearing documents according to any time limits set out in the order from the tribunal.<br><br>Refer to the orders from the tribunal about whether you need to include your witness statements here or separately.",
+    "Label": "Please refer to any instructions or guidance given by the Tribunal as to what documents are needed for the hearing.<br><br>These documents, usually agreed by both parties, will form part of the evidence at the hearing. You should include all the documents that are important to your case, and that both parties want to refer to at the hearing.<br><br>The combined set of documents must be in one PDF file, usually submitted by one party unless instructed otherwise by the tribunal.<br> <br>Generally the documents should be included in the PDF file in date order, and the first page or pages of the pdf numbering should be an index showing the name of the document, its date and the page number.<br><br>The single PDF document should be created offline before uploading through this form.<br><br>You must submit hearing documents according to any time limits set out by the Tribunal.<br><br>Refer to the orders from the tribunal about whether you need to include your witness statements here or separately.",
     "FieldType": "Label",
     "SecurityClassification": "Public"
   },
   {
     "CaseTypeID": "ET_EnglandWales",
     "ID": "bundlesRespondentPrepareDocNotes2",
-    "Label": "Providing an appropriate set of documents will ensure your case gets a fairer hearing and limit delays for you and the tribunal.",
+    "Label": "Providing a file which contains all relevant documents, and only relevant documents, will help ensure a fair hearing and minimise any delays.",
```

Comment:
I understand this is likely mandated in the ticket. But this sounds more link HintText to me?